settings: Update wording for PM settings on notifications tab.

rishig opened the pull request:
Some minor cleanup.

![image](https://user-images.githubusercontent.com/890911/56250867-02b72b00-6066-11e9-8f4f-9f792f47013e.png)

timabbott commented on static/js/settings.js:
```diff
@@ -94,9 +94,9 @@ function setup_settings_label() {
-        enable_online_push_notifications: i18n.t("Mobile notifications always (even when online)"),
+        enable_offline_email_notifications: i18n.t("Email notifications"),
+        enable_offline_push_notifications: i18n.t("Mobile notifications"),
+        enable_online_push_notifications: i18n.t("Mobile notifications for every message, including ones I read"),
```
I don't think this is an accurate description of the behavior -- it doesn't matter whether you read the messages, what matters is whether you're active (e.g. we check whether you have a connected client and that your UserPresence is recent).

rishig replied:
I thought we send a push notification as long as the message is off screen?

timabbott replied:
No -- the backend code that determines whether to send a push notification has no access to information about what view the user is looking at.  We send a **desktop notification** as long as the message is offscreen.  But the determination for whether to send a push/email notification is based on whether the user is likely to get a desktop notification, i.e. whether the user was present within the last few minutes on a client.

rishig commented:
Updated, and added a few more commits (as a precursor to getting to https://chat.zulip.org/user_uploads/2/b4/v7goeE5YkCpP5E7i0aOCjAXQ/pasted_image.png).

The vertical spacing is pretty bad, but I think will require a deeper fix. 
New:
![image](https://user-images.githubusercontent.com/890911/56447555-d0a00600-62bd-11e9-91fe-3a0c23ca7694.png)

Old:
![image](https://user-images.githubusercontent.com/890911/56447584-f4634c00-62bd-11e9-9063-8c453fecadaa.png)

timabbott commented:
Merged, thanks @rishig!

timabbott commented:
I also closed #10381 and opened https://github.com/zulip/zulip/issues/12198 based on working on this.  I assume you'll do whatever update to #12182 you had had in mind.

rishig commented:
cool, will do.
